src/index.ts

```diff
@@ -35,6 +36,9 @@ export {
     connect,
     $do,
     types as coreActions,
+    ActionPayloadTypes as CoreActionPayloadTypes,
+    AnyAction as CoreAnyAction,
```

Comment:
Why are we renaming them? Client application can set the names for themselves during the import